pandas/core/groupby/groupby.py

```diff
@@ -4086,7 +4086,10 @@ def _post_process_cython_aggregate(self, obj):
     def aggregate(self, arg, *args, **kwargs):
 
         _level = kwargs.pop('_level', None)
-        result, how = self._aggregate(arg, _level=_level, *args, **kwargs)
+        _agg_kwargs = kwargs.copy()
```

Comment:
you can just list axis as a kwarg

Reply:
That breaks a test. The issue if ``axis`` in is in ``_kwargs``, you may get (from the breaking test):

```python
>>> _level, args, _agg_kwargs = None, (80,), {'axis': 0}
>>> self._aggregate(arg, _level=_level, *args, **_agg_kwargs)
TypeError: _aggregate() got multiple values for argument 'axis'
```

Reply:
Not sure I understand this - what test is breaking? Perhaps the test is configured incorrectly?

Reply:
Both ``80`` and ``0`` may be the value for parameter ``axis``. The signature is ``def _aggregate(self, arg, axis=0, *args, **kwargs)``, so the second unnamed argument (``80``) will be considered to be ``axis``, but this clashes with the parameter in kwargs (``{'axis': 0}``), causing the exception.

To avoid this we'd prefer the signature to be ``def _aggregate(self, arg, *args, axis=0, **kwargs)``, but this syntax is only supported in Python3...

Reply:
Where is that test located?

Reply:
``pandas\tests\groupby\test_groupby.py::test_pass_args_kwargs``